rules/Php70/EregToPcreTransformer.php

```diff
@@ -71,19 +71,8 @@ public function __construct(
     ) {
```

Comment:
I wonder why the delimiter is a config param and not auto-detected?

Its pretty unlikely that all reges in a given codebase use the same delimiter all over the placd

Reply:
it seems possibly was exists since configurable rector not yet a think, so need to configure via services, I don't change it because it seems not matter since ereg is not using delimiter per documentation, so on transform, using `#` as delimiter after transformed to preg_match seems safe